### PR TITLE
Math magicians hooks

### PR DESCRIPTION
**Pull request by brainconnect93**
I did this project by refactoring the class components using React hooks

### The following setups were implemented

- Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.
- After the refactor is done, test the app in the browser, and it keeps the same functionality.
- Commit my Changes 
- Push to the Repo

### CODE REVIEW

**Review by zhadier — CHANGES_REQUESTED** [review]
# Hi @brainconnect93  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 



## Highlights
✔️ No linter errors
✔️ GitFlow was set up correctly
✔️ Professional pull request title and description
✔️ Your design looks 🔥 

## Required Changes ♻️

- [ ] Kindly make the App.js a functional component as well, since the type of component you follow usually has to be uniform throughout the project.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,50 +1,42 @@
...
-              <button type="button" className="calc active" onClick={this.handleClick}>-</button>
-              <button type="button" className="calc" onClick={this.handleClick}>1</button>
-              <button type="button" className="calc" onClick={this.handleClick}>2</button>
-              <button type="button" className="calc" onClick={this.handleClick}>3</button>
-              <button type="button" className="calc active" onClick={this.handleClick}>+</button>
-              <button type="button" className="zero" onClick={this.handleClick}>0</button>
-              <button type="button" className="calc" onClick={this.handleClick}>.</button>
-              <button type="button" className="calc active" onClick={this.handleClick}>=</button>
-            </div>
+function Calculator() {
```
Comment (keneogu) [inline src/components/calculator.js]: Your component should use the es6 arrow function
`const Calculator = () => (
      <div>
       // Calculator app design goes here
      </div>
);

export default Calculator;
`

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @brainconnect93,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Nice design
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (brainconnect93) [reply src/components/calculator.js]: done

**Review by keneogu — APPROVED** [review]
Hi @brainconnect93,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**